extensions/blocks/podcast-player/components/audio-player.js

```diff
@@ -0,0 +1,72 @@
+/* global _wpmejsSettings, MediaElementPlayer */
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+
+const meJsSettings = typeof _wpmejsSettings !== 'undefined' ? _wpmejsSettings : {};
+
+class AudioPlayer extends Component {
+	audioRef = el => {
+		if ( el ) {
+			// Construct audio element.
+			this.audio = document.createElement( 'audio' );
+			this.audio.src = this.props.initialTrackSource;
+			this.audio.addEventListener( 'play', this.props.handlePlay );
+			this.audio.addEventListener( 'pause', this.props.handlePause );
+			this.audio.addEventListener( 'error', this.props.handleError );
+
+			// Insert player into the DOM.
+			el.appendChild( this.audio );
+
+			// Initialize MediaElement.js
+			this.mediaElement = new MediaElementPlayer( this.audio, meJsSettings );
+		} else {
+			// Cleanup.
+			this.mediaElement.remove();
+		}
+	};
+
+	/**
+	 * Play current audio.
+	 * @public
+	 */
+	play = () => {
+		// Ignoring exceptions as they are handled globally from the audio element.
```

Comment:
> handled globally

In what sense? I wasn't entirely clear.

Reply:
We listen for error events on the audio element.
https://github.com/Automattic/jetpack/pull/15072/files/f21cf726b28922d372e7cfbc7d497cb6aface16f#diff-df8fb697659055766a98914a6a2c68b8R21